### PR TITLE
Here we go again

### LINKED ISSUE #818: "Ah shit, here we go again" generator
meme template

### PR DESCRIPTION
**Pull request by CamHardy**
#818 is finally done

Closes #818

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1593,7 +1593,7 @@
       "from": "github:Doges/crowdin-node",
       "requires": {
         "bluebird": "^3.7.2",
-        "graceful-fs": "^4.2.4",
+        "graceful-fs": "^4.2.2",
```
Comment (almeidx) [inline package-lock.json:1596]: ?

File: src/commands/memes/ashwga.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const { CanvasTemplates, Command } = require('../../')
+
+const { MessageAttachment } = require('discord.js')
+
+module.exports = class HereWeGoAgain extends Command {
+  constructor (client) {
+    super({
+      name: 'herewegoagain',
+      aliases: ['ahshit', 'ashwga', 'hwga'],
```
Comment (almeidx) [inline src/commands/memes/ashwga.js:9]: perhaps add `ahshitherewegoagain`

**Review by almeidx — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (CamHardy) [reply package-lock.json:1596]: Whoops this wasn't intentional, maybe it was auto-edited when I ran `npm i`?

Comment (almeidx) [reply package-lock.json:1596]: Maybe. You should run `npm ci` instead of `npm i`

File: src/commands/memes/ashwga.js (hunk 2 above)
Comment (CamHardy) [reply src/commands/memes/ashwga.js]: Added!

File: package-lock.json (hunk 1 above)
Comment (CamHardy) [reply package-lock.json:1596]: First time I've heard of `npm ci`. Cool, I'll use that from now on!

Comment (almeidx) [reply package-lock.json:1596]: if you could revert this change that would be nice

**Comment by almeidx** [comment]
wrong line ;)

**Review by almeidx — APPROVED** [review]

**Review by Doges — APPROVED** [review]
LGTM

**Pull request merged**